### PR TITLE
Vertx 3.7.0 update

### PR DESCRIPTION
**Pull request by vietj**
Set of fixes for 3.7.0

Note the Reactive PG Client has been disabled from the build as it does not yet implement the new `ReadStream#pipe` method that has been introduced in 3.7.0

When the Reactive PG Client will be updated, it can be re enabled again.

### CODE REVIEW

File: vertx-axle-clients/vertx-axle-generator/src/test/java/io/vertx/core/streams/package-info.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2011-2017 Contributors to the Eclipse Foundation
```
Comment (cescoffier) [inline vertx-axle-clients/vertx-axle-generator/src/test/java/io/vertx/core/streams/package-info.java]: Not to be fixed here, but something to update in codegen.

Comment (vietj) [reply vertx-axle-clients/vertx-axle-generator/src/test/java/io/vertx/core/streams/package-info.java]: those are test classes and are copied this way.

I think we should add this repository in .genignore and remove the classes (in another PR)

**Comment by vietj** [comment]
I've updated with latest PG client version, that was just released in central but likely not yet there

**Comment by vietj** [comment]
I removed the copied classes and added to .gitignore file

**Pull request merged**